src/types/video.ts

```diff
@@ -240,12 +255,13 @@ export interface ReactVideoProps extends ReactVideoEvents, ViewProps {
   pictureInPicture?: boolean; // iOS
   playInBackground?: boolean;
   playWhenInactive?: boolean; // iOS
-  poster?: string;
-  posterResizeMode?: EnumValues<PosterResizeModeType>;
+  poster?: string | ReactVideoPoster; // string is deprecated
```

Comment:
can you also mark deprescated with /** @deprecated */ like in this commit: https://github.com/TheWidlarzGroup/react-native-video/pull/3867/commits/731d1e77b18a388c12823eab711b117a6fce5e1e

Reply:
at `poster` it's impossible to handle deprecated only for one type. I changed it for `posterResizeMode`